File: Python/principal_conta.py
from conta import Conta

conta1 = Conta("001", "12345-6")
conta2 = Conta("002", "98765-4")

print(f"Conta 1: Banco {conta1.get_banco()}, Número {conta1.get_numero()}")
print(f"Conta 2: Banco {conta2.get_banco()}, Número {conta2.get_numero()}")

conta2.transferir(100, conta1, None)

conta2.transferir(50, "002", "98765-4")

File: basico/Java/PrincipalConta.java
public class PrincipalConta {
    public static void main(String[] args) {
        Conta conta1 = new Conta(123, "456789-0");
        Conta conta2 = new Conta(456, "987654-3");

        System.out.println("Conta 1: Banco " + conta1.getBanco() + ", Número " + conta1.getNumero());
        System.out.println("Conta 2: Banco " + conta2.getBanco() + ", Número " + conta2.getNumero());

        conta1.transferir(456, "987654-3", 100.00);
        
        conta1.transferir(conta1, 200.00);
    }
}
